.pre-commit-config.yaml

```diff
@@ -31,5 +31,5 @@ repos:
     hooks:
       - id: black
         additional_dependencies: ['.[jupyter]']
-        types: [python, jupyter]
-        types_or: [python, jupyter]
+        types: [file]
+        files: '\.(py|ipynb)$'
```

Comment:
Use `- id: black-jupyter`. It will do this trick for you.

See:

- https://github.com/sgl-project/sglang/pull/1938#discussion_r1831071218